src/api/index.ts

```diff
@@ -259,7 +260,7 @@ export async function upgrade(input: Input): Promise<Output> {
           return m.name === `${p1Model.name}_${p1Field.name}`
         })
         if (p2ModelList) {
-          p2ModelList.remove()
+          // p2ModelList.remove()
```

Comment:
These models are needed until you apply breaking changes for scalar lists (if you're in Postgres)